src/guide/Join.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Join the Vue Community!
+type: guide
+order: 20
+---
+
+## Join the Vue.js Community!
+
+The vibrant community around Vue.js is continually growing. Nevertheless, helpful hands are always welcome. In fact, a growing community is the best sign of a healthy OSS project worth putting any effort into, even if it is as the user of the software. 
+
+If you would like to also be a part of the Vue.js community, this page was meant to support your endeavors. Have a look at the resources listed below to get some orientation around the sailing ship of Vue.js and...
+
+....welcome aboard!
+
+## Resources You Should Now About!
+
+### The Community
+- [The Forum](http://forum.vuejs.org/) - THE best place to ask questions and get answers about Vue.js and its components.
+- [Gitter Channel](https://gitter.im/vuejs/vue) -  A place for devs to meet and chat. You can ask questions here, but the forum is the better platform, since the discussions are threaded.
+- [Github](https://github.com/vuejs) - Report bugs or fork repositories to offer pull requests. Any contribution is greatly appreciated!  
+
+
+### Third Party Contributions
+
+- [The Awesome Vue Page](https://github.com/vuejs/awesome-vue) - See what other awesome resources have been published by other awesome people.
+- [The "Show and Tell" Subforum](http://forum.vuejs.org/category/15/show-tell) - Another great place to check out what others have built with and for the growing Vue ecosystem.
+ 
+### Contributing to Vue
+
+- [The rules to contribute](https://github.com/vuejs/vue/blob/dev/CONTRIBUTING.md) - As with any community, there has to be rules, in order to ensure everyone is doing the right things the right way.
+
+- The Main Components of Vue
+  - [Vue](https://github.com/vuejs/vue) - The core repository.
+  - [Vuex](https://github.com/vuejs/vuex) - A Flux-inspired Application Architecture for VueJS.

```

Comment:
Yeah, I noticed that and actually thought I corrected them all. I was pretty tired when I wrote it. 

Corrections are done. :smile: 

Scott